snap/backend/snap.manifest.json

```diff
@@ -1,18 +1,18 @@
 {
   "version": "0.1.0",
-  "description": "Symbol snap.",
+  "description": "Manage Symbol mosaics, and connect to Symbol apps with MetaMask.",
```

Comment:
What Symbol apps can be connected to? 🤔

Reply:
In my vision, in the future it may have a lot of different Symbol apps can be create.
So in example, the wallet (frontend) can consider as Symbol apps.

In future we can create many different apps, such as a website can sell NFT.
The apps will asking `snap` transfer fund to creators as purchase NFT.